docs/toml.md

```diff
@@ -455,6 +455,7 @@ defaultEntryPoints = ["http", "https"]
   backend = "backend1"
   passHostHeader = true
   priority = 10
+  whitelistSourceRange = ["10.42.0.0/16", "152.89.1.33/32", "afed:be44::/16"]
```

Comment:
I suppose a bit of extra prose might be nice that explains things like

- what does this feature do;
- how does it behave (including error cases); and
- what providers are supported.